### PR TITLE
FEATURE: [bybit] add balance snapshot event

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

**Review by ycdesu — APPROVED** [review]
Nice

File: pkg/exchange/bybit/stream.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ type Stream struct {
 	key, secret string
 	types.StandardStream
 
-	bookEventCallbacks []func(e BookEvent)
+	bookEventCallbacks   []func(e BookEvent)
+	walletEventCallbacks []func(e []*WalletEvent)
```
Comment (c9s) [inline pkg/exchange/bybit/stream.go:36]: use slice of value?

Comment (c9s) [reply pkg/exchange/bybit/stream.go:36]: oh it's a very large struct, ok let's use pointer LOL

**Review by c9s — APPROVED** [review]

File: pkg/exchange/bybit/stream.go (hunk 1 above)
Comment (bailantaotao) [reply pkg/exchange/bybit/stream.go:36]: Yeah~ that's why I chose him in the end

**Pull request merged**